### ISSUE TITLE
Driveitem: Checkin() and Checkout() actions not working

### ISSUE THREAD

**Issue by mariusstaub** [microsoftgraph/msgraph-sdk-dotnet#628]
According to the [documentation](https://docs.microsoft.com/en-us/graph/api/driveitem-checkin?view=graph-rest-beta&tabs=http) the `/checkin` and `/checkout` actions are still in beta. However when I use the Microsoft.Graph Nuget package 1.21.0 (not beta) I can reach these actions. `.Checkin()` returns "The file is not checked out." and `.Checkout()` returns "Unknown Error".

**To Reproduce**
```
var graphClient = new GraphServiceClient(...);
await graphClient.Drives["drive-id"].Items["file-id"].Checkout().Request().PostAsync();
```
or
`await graphClient.Drives["drive-id"].Items["file-id"].Checkin().Request().PostAsync();`

**Expected behavior**
I expect either:
a) The api is still in beta => the `.Checkin()` and `.Checkout()` actions should not be available or
b) The api is 1.0 => the `.Checkout()` action should work and not return 'Unknown error'

**Desktop**
- OS: Windows 10 
- Microsoft.Graph 1.21.0

**Additional context**
The same issue exists in Graph Explorer: `/checkin` and` /checkout` are available in `/v1.0` but `/checkout` does not work.
 
[AB#7269](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7269)

**Comment by MIchaelMainer** [comment]
It is possible that this is unintentionally in the v1.0 endpoint. The documentation is not present for v1.0 which gives me the impression that it shouldn't be there. 

Does this still work in /beta?

**Comment by arsnyder16** [comment]
@mariusstaub @MIchaelMainer  I am successfully using checkin/checkout from the javascript sdk. I haven't tried the .NET sdk but the js sdk uses `/v1.0/` endpoint contrary to what the documentation says.   Be aware that the check out/check in isn't as complete as it was in sharepoint the graph api is lacking a way to discard checkouts.  So even if you do not make changes to the file there is no way to prevent extra versions being added since you cannot discard.

**Comment by andrueastman** [comment]
@mariusstaub According to the docs it looks like this api is now available in v1.0 as well.
https://docs.microsoft.com/en-us/graph/api/driveitem-checkin?view=graph-rest-1.0&tabs=http
https://docs.microsoft.com/en-us/graph/api/driveitem-checkout?view=graph-rest-1.0&tabs=http

We will close this issue for now. Feel free to reopen it if there are any issues with this.
